.doc_gen/metadata/sqs_metadata.yaml

```diff
@@ -965,6 +974,15 @@ sqs_DeadLetterQueue:
               snippet_tags:
                 - cpp.example_code.sqs.SetDeadLetterQueue.config
                 - cpp.example_code.sqs.SetDeadLetterQueue
+    JavaScript:

```

Comment:
This looks like a new snippet not tagged before, should it also be added to the README here: https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/javascriptv3/example_code/sqs/README.md ?

Reply:
Good catch. I'll run a writeme